### PR TITLE
Renewal of shop purchase page

### PR DESCRIPTION
**Pull request by hskim881028**
### Description

Renewal of shop purchase page

```Focus on the list, please review``` :pray:
**nekoyume/Assets/_Scripts/State/ReactiveShopState.cs** 
https://github.com/planetarium/NineChronicles/pull/89/files#diff-2e0bf62cb3510649e248e062919b9c65ba6e1caeee3d636e633502c37599304d

**nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs**
https://github.com/planetarium/NineChronicles/pull/89/files#diff-69bc228a7e488d68e730931b3afb9790b5726c653cf8389c20b8e047b66fdc0e

**nekoyume/Assets/_Scripts/BlockChain/ActionManager.cs**
https://github.com/planetarium/NineChronicles/pull/89/files#diff-6ab4b4aa801373d7aaa26fed671faeeeeff4a1392e3bfefe1120bed18c9c4e32

**nekoyume/Assets/_Scripts/UI/Shop/ShopBuy.cs**
https://github.com/planetarium/NineChronicles/pull/89/files#diff-d5ffac9196785cf9f943f222fa0b825e7f6e06b079abed971888587562754d44

**nekoyume/Assets/_Scripts/UI/Shop/ShopBuyBoard.cs**
https://github.com/planetarium/NineChronicles/pull/89/files#diff-839bb884f64885c835fb065e0c724504ad95c3a4c38715f3b3dc480a6173be1b

**nekoyume/Assets/_Scripts/UI/Model/ShopItems.cs** 
https://github.com/planetarium/NineChronicles/pull/89/files#diff-174011517290b9265b0943fc0be9f9116d5adfd9aa83f8e789cadeb9a027c660

**nekoyume/Assets/_Scripts/UI/Shop/ShopSell.cs**
https://github.com/planetarium/NineChronicles/pull/89/files#diff-10a924ad8d653f6ef102104c80fa357d10ca234f420ccd5d30f9d50a48fe8cac

### Required Reviewers

@planetarium/9c-dev , @ipdae , @Namyujeong

### CODE REVIEW

File: nekoyume/Assets/_Scripts/BlockChain/Agent.cs [hunk 1]
```diff
@@ -418,10 +418,6 @@ private void InitAgent(Action<bool> callback, PrivateKey privateKey, CommandLine
                 }
 
                 // 상점의 상태를 한 번 동기화 한다.
-                States.Instance.SetShopState(
```
Comment (longfin) [inline nekoyume/Assets/_Scripts/BlockChain/Agent.cs:421]: Is it okay that we remove it?

File: nekoyume/Assets/_Scripts/BlockChain/RPCAgent.cs [hunk 2]
```diff
@@ -209,10 +209,6 @@ private IEnumerator CoJoin(Action<bool> callback)
             }
 
             // 상점의 상태를 한 번 동기화 한다.
-            States.Instance.SetShopState(
```
Comment (longfin) [inline nekoyume/Assets/_Scripts/BlockChain/RPCAgent.cs:212]: Is it okay that we remove it?

Comment (hskim881028) [reply nekoyume/Assets/_Scripts/BlockChain/RPCAgent.cs:212]: > Is it okay that we remove it?

It calls `SetShopState()` every time you enter the shop, so it doesn't seem necessary here. :thinking_face:

File: nekoyume/Assets/_Scripts/BlockChain/Agent.cs (hunk 1 above)
Comment (hskim881028) [reply nekoyume/Assets/_Scripts/BlockChain/Agent.cs:421]: > Is it okay that we remove it?

It calls `SetShopState()` every time you enter the shop, so it doesn't seem necessary here. :thinking_face:

**Review by longfin — APPROVED** [review]

**Pull request merged**